### PR TITLE
Fix deployment HPC-61616

### PR DESCRIPTION
**Pull request by itkovian**
- the options were not passed along correctly
    - list had an int instead of a string
    - two arguments were passed in a single list entry

tcnhw. ever.

### CODE REVIEW

File: lib/vsc/filesystem/gpfs.py [hunk 1]
```diff
@@ -483,11 +483,11 @@ def _list_disk_single_device(self, device):
         # if this fails, nodes probably have shortnames
         try:
             # - means disk offline, so no nodename
-            alldomains = ['.'.join(x.split('.')[1:]) for x in infoM['IOPerformedOnNode'] if not x in ['-', 'localhost']]
+            alldomains = ['.'.join(x.split('.')[1:]) for x in infoM['IOPerformedOnNode'] if x not in ['-', 'localhost']]
             if len(set(alldomains)) > 1:
                 self.log.error("More then one domain found: %s." % alldomains)
             commondomain = alldomains[0]  # TODO: should be most frequent one
-        except:
+        except KeyError:
```
Comment (wpoely86) [inline lib/vsc/filesystem/gpfs.py]: `IndexError` too?

File: lib/vsc/filesystem/gpfs.py [hunk 2]
```diff
@@ -629,6 +629,7 @@ def make_fileset(self, new_fileset_path, fileset_name=None, parent_fileset_name=
         - create fileset in existing fileset
         - create fileset with fsetpath part of symlink
         """
+        del afm
```
Comment (wpoely86) [inline lib/vsc/filesystem/gpfs.py]: unused? Just tell pylint it should ignore it.

File: lib/vsc/filesystem/gpfs.py [hunk 3]
```diff
@@ -858,20 +859,19 @@ def _set_quota(self, soft, who, obj=None, typ='user', hard=None):
             self.raiseException("setQuota: can't set hard limit %s lower then soft limit %s" %
                                 (hard, soft), GpfsOperationError)
 
-        opts += ["-%s" % typ2opt[typ], who]
+        opts += ["-%s" % typ2opt[typ], "%s" % who]
```
Comment (wpoely86) [inline lib/vsc/filesystem/gpfs.py]: isn't `str(who)` cleaner?

File: lib/vsc/filesystem/gpfs.py (hunk 2 above)
Comment (itkovian) [inline lib/vsc/filesystem/gpfs.py]: please agree with the others since they thought differently :)

File: lib/vsc/filesystem/gpfs.py (hunk 3 above)
Comment (itkovian) [inline lib/vsc/filesystem/gpfs.py]: It's the same.

**Pull request merged**